test/unit/cssTest.js

```diff
@@ -0,0 +1,268 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const path = require('path')
+const generateTestApp = require('../util/generateTestApp')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const fork = require('child_process').fork
+const fs = require('fs')
+const fse = require('fs-extra')
+const klawsync = require('klaw-sync')
+
+// appDir
+const appDir = path.join(__dirname, '../', 'app', 'cssTest')
+
+// basic CSS files
+let cssDataArray = [
+  `body { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`,
+  `#selector { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`,
+  `.body { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`
+]
+
+// version package json file
+let packageJSON = `{
+ "version": "0.3.1",
+ "rooseveltConfig": {}
+}`
+
+// version num
+let verNumber = '0.3.1'
```

Comment:
This separate variable for version number is not needed when you use an object for `packageJSON`